content/en/docs/ops/setup/standalone-operator/index.md

```diff
@@ -1,196 +1,134 @@
 ---
-title: Standalone Operator Quick Start Evaluation Install [Experimental]
-description: Instructions to install Istio in a Kubernetes cluster for evaluation.
+title: Standalone Operator Install [Experimental]
+description: Instructions to install Istio in a Kubernetes cluster using the Istio operator.
 weight: 11
 keywords: [kubernetes, operator]
 aliases:
 ---
 
-This guide installs Istio using the standalone Istio operator. The only dependencies
-required are a supported Kubernetes cluster and the `kubectl` command. This
-installation method lets you quickly evaluate Istio in a Kubernetes cluster on
-any platform using a variety of profiles.
+This guide installs Istio using the standalone Istio
+[operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+The only dependencies required are a supported Kubernetes cluster and the `kubectl` command.
 
-To install Istio for production use, we recommend using the [Helm Installation guide](/docs/setup/install/helm/)
+{{< warning >}}
+To install Istio for production use, we recommend [installing with {{< istioctl >}}](/docs/setup/install/istioctl/)
 instead, which is a stable feature.
+{{< /warning >}}
 
 ## Prerequisites
 
 1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
 
 1. Check the [Requirements for Pods and Services](/docs/setup/additional-setup/requirements/).
 
-## Installation steps
-
-1. Install Istio using the operator with the demo profile:
-
-    {{< text bash >}}
-    $ kubectl apply -f https://preliminary.istio.io/operator.yaml
-    {{< /text >}}
-
-{{< warning >}}
-This profile is only for demo usage and should not be used in production.
-{{< /warning >}}
-
-1. (Optionally) change profiles from the demo profile to one of the following profiles:
+## Install
 
-{{< tabset cookie-name="profile" >}}
-
-{{< tab name="demo" cookie-value="permissive" >}}
-When using the [permissive mutual TLS mode](/docs/concepts/security/#permissive-mode), all services accept both plaintext and
-mutual TLS traffic. Clients send plaintext traffic unless configured for
-[mutual TLS migration](/docs/tasks/security/mtls-migration/). This profile is installed during the first step.
-
-Choose this profile for:
-
-* Clusters with existing applications, or
-* Applications where services with an Istio sidecar need to be able to
-  communicate with other non-Istio Kubernetes services
-
-Run the following command to switch to this profile:
+To install the Istio `demo` [configuration profile](/docs/setup/additional-setup/config-profiles/)
+using the operator, run the following command:
 
 {{< text bash >}}
-$ kubectl apply -f https://preliminary.istio.io/operator-profile-demo.yaml
+$ kubectl apply -f https://preliminary.istio.io/operator.yaml
```

Comment:
I worry that this install is prone to races - at least one tester experienced that. @sdake could we change this back to using kubectl -k? Otherwise we need to install the CRD separately.

Reply:
IMO it would be better if operator.yaml just deployed the controller. Then we apply the CR in a second command. The single command install is not helpful in this guide, the intent of which is to show people how to use the operator.

Reply:
Not CR, the CRD. If that is not created first, the manifest apply can error out. 
If we separate the CR, it would be three apply commands.

Reply:
We still would have to separate the CRD, so it would be three apply commands.

Reply:
Won't we avoid the race if we just move the CR to a second apply? Aren't the CRDs only needed when you try to apply the CR, so it won't matter what order they're created in operator.yaml?

Reply:
Right, but the CR can still race with the CRD.

Reply:
Per our chat, it's ok to have everything in one base file and a separate CR. I've changed things around to support that in this PR: 
https://github.com/istio/istio.io/pull/5436
We should also update the blog to reflect these changes.

Reply:
@ostromart the CRD need not be separated.  I'd separate thee CRs, not the CRDs.